### PR TITLE
28442 Debt resolution codeownership

### PR DESCRIPTION
**Pull request by dillo**
## Description of change

- Codeowner ship setup for the VSA Debt Resolution team.
- Github team is @department-of-veterans-affairs/vsa-debt-resolution
## Original issue(s)
department-of-veterans-affairs/va.gov-team#28442

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -47,4 +47,12 @@ spec/support/*/facilities @department-of-veterans-affairs/vsa-facilities-backend
 spec/support/*/lighthouse @department-of-veterans-affairs/vsa-facilities-backend @department-of-veterans-affairs/backend-review-group
 
 # test user dashboard product
-modules/test_user_dashboard @department-of-veterans-affairs/vsp-identity
```
Comment (LindseySaari) [inline .github/CODEOWNERS:50]: Did the trailing newline get removed here?

Comment (LindseySaari) [reply .github/CODEOWNERS:50]: I see "no newline at the end of file"

Comment (dillo) [reply .github/CODEOWNERS:50]: I don't think the original had a new line at the end of the file, to begin with. I turned off that feature on my editor and saved and committed it again without this new line.

Comment (dillo) [reply .github/CODEOWNERS:50]: Maybe the CODEOWNERS file is setup to be handled in this fashion?

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**